app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityTest.kt

```diff
@@ -271,6 +272,72 @@ class NavigationDrawerActivityTest {
     }
   }
 
+  @Test
+  fun testNavigationDrawerTestActivity_openNavDrawer_clickSwitchProfile_clickCancel() {
+    launch<NavigationDrawerTestActivity>(
+      createNavigationDrawerActivityIntent(internalProfileId)
+    ).use {
+      it.openNavigationDrawer()
+      onView(withText(R.string.menu_switch_profile)).perform(click())
+      onView(withText(R.string.home_activity_back_dialog_cancel))
+        .inRoot(isDialog())
+        .perform(click())
+      it.openNavigationDrawer()
+      onView(withId(R.id.nav_home)).check(matches(ViewMatchers.isChecked()))
```

Comment:
@anandwana001 On line 286, we are checking if the view with text `Home` is checked or not, but it is also considering the textview with text `Home` in the Actionbar of the HomeActivity.
![IMG_20201224_145914](https://user-images.githubusercontent.com/54636525/103079164-5e86d500-45f9-11eb-96e7-a6daf4a5a474.jpg)
![IMG_20201224_145921](https://user-images.githubusercontent.com/54636525/103079177-621a5c00-45f9-11eb-8c83-7db84d2591b8.jpg)